depthai_nodes/ml/parsers/utils/yolo.py

```diff
@@ -173,27 +189,46 @@ def non_max_suppression(
 def parse_yolo_outputs(
     outputs: List[np.ndarray],
     strides: List[int],
-    anchors: np.ndarray,
+    no: int,
+    anchors: Optional[List[np.ndarray]] = None,
     kpts: Optional[List[np.ndarray]] = None,
+    det_mode: bool = False,
+    yolo_version: YOLOVersion = YOLOVersion.DEFAULT,
 ) -> np.ndarray:
     """Parse all outputs of an YOLO model (all channels).
 
     @param outputs: List of outputs of an YOLO model.
     @type outputs: List[np.ndarray]
     @param strides: List of strides.
     @type strides: List[int]
-    @param anchors: List of anchors.
-    @type anchors: np.ndarray
+    @param no: Number of outputs of the model.
```

Comment:
can we have better variable name? `num_outputs` maybe?

Reply:
Thanks for the suggestion, I have made the change accordingly.
Btw I had just kept that variable name as `no` mostly to be compatible with ultralytics way of naming this variable, but I agree it should be more descriptive.